pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1351,8 +1351,8 @@ let
           # semver scheme, contrary to preview versions which are listed on
           # the VSCode Marketplace and use a calver scheme. We should avoid
           # using preview versions, because they expire after two weeks.
-          version = "14.1.1";
-          sha256 = "sha256-eSN48IudpHYzT4u+S4b2I2pyEPyOwBCSL49awT/mzEE=";
+          version = "14.8.2";
```

```suggestion
          version = "14.9.0";
```

Please update meta.description:

> description = "A Visual Studio Code extension for extending git capabilities";

Feel free to improve the description. But try to keep the format of no '.' in the end. And starting with 'A Visual Studio Code extension ...'